sklearn/utils/estimator_checks.py

```diff
@@ -131,7 +131,8 @@ def _yield_transformer_checks(name, Transformer):
                     'PLSCanonical', 'PLSRegression', 'CCA', 'PLSSVD']:
         yield check_transformer_data_not_an_array
     # these don't actually fit the data, so don't raise errors
-    if name not in ['AdditiveChi2Sampler', 'Binarizer', 'Normalizer']:
+    if name not in ['AdditiveChi2Sampler', 'Binarizer',
```

Comment:
sklearn.tests.test_common.test_non_meta_estimators

Removing these causes errors in `sklearn.tests.test_common.test_non_meta_estimators`